bash/upload.bash

```diff
@@ -93,16 +96,32 @@ _update() {
 }
 
 ###################################################
-# Print the contents of info file if scipt is installed system wide.
-# Path is INFO_FILE="${HOME}/.google-drive-upload/google-drive-upload.info"
-# Globals: 1 variable
-#   HOME
+# Update in script values
+###################################################
+_update_value() {
+    declare command_path="${INSTALL_PATH:?}/${COMMAND}" \
+        value_name="${1:-}" value="${2:-}" \
+        script_without_value_and_shebang="$(grep -v "${value_name}=\".*\".* # added values" "${command_path}" | sed 1d)"
```

Comment:
Codacy found an issue: [Declare and assign separately to avoid masking return values.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6201392)